brave/brave-core
# Remove auto increment from contribution_queue

Resolves https://github.com/brave/brave-browser/issues/8804

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Create new profile
- Enable Rewards
- Ensure that `contribution_queue` table id column is of type `TEXT` in database
- Ensure that `contribution_queue_publishers` table id column is of type `TEXT` in database
- Tip a publisher and ensure that it is processed and summary looks as expected

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
